cmd/monaco/deploy/deploy.go

```diff
@@ -287,6 +287,10 @@ func validateAuthenticationWithProjectConfigs(projects []project.Project, enviro
 							return fmt.Errorf("API of type '%s' requires a token for environment '%s'", conf.Type, envName)
 						}
 					case config.SettingsType:
+						t, ok := conf.Type.(config.SettingsType)
+						if ok && t.AllUserPermission != nil && environments[envName].Auth.OAuth == nil {
+							return fmt.Errorf("Using permission property on settings API requires OAuth, schema '%s' enviroment '%s'", t.SchemaId, envName)
```

Comment:
Missed that one. It should start lower case right? ON the other hand we have the `API` uppercase in the other errors

Reply:
I got confused, about why others start uppercase. I see now its the API acronym :D. Thanks for pointing it out 👍